tests/test_flows_seed.py

```diff
@@ -476,6 +476,26 @@ def load_right_seed_into_decoder(view: View):
             FlowStep(seed_views.SeedOptionsView),
         ])
 
+    # TODO: ideally this test should be part of controller related test? move it there
```

Comment:
I think it's fine to leave this flow test here to explicitly test that the screensaver cannot start during the interactive transcription process.